### PR TITLE
Added compat data for management.install() method.

### PR DESCRIPTION
**Pull request by irenesmith**

### CODE REVIEW

**Comment by irenesmith** [comment]
I added a note as follows:
   "Only extensions whose 'type' is 'theme' can be installed and enabled using this method."

Also, for the record, this applies to work done for [1369209](https://bugzilla.mozilla.org/show_bug.cgi?id=1369209)

File: webextensions/api/management.json [hunk 1]
```diff
@@ -403,6 +403,33 @@
                 }
               }
             }
+          },
+          "install": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": false
```
Comment (jpmedley) [inline webextensions/api/management.json]: @Elchi3 If Android and Webview are missing, will the rendered table have 'No' or '?'? I've forgotten.

Comment (Elchi3) [reply webextensions/api/management.json]: Question mark

**Comment by wbamberg** [comment]
> I added a note as follows:
"Only extensions whose 'type' is 'theme' can be installed and enabled using this method."

I think this is a little unclear. It should refer to a specific object, or I'm left wondering where to look up `type`. In this case I guess it's [`ExtensionInfo.type`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/management/ExtensionInfo).

Also though, (and unfortunately I think we are not very consistent with terminology here) it seems that we usually use "Add-ons" as the generic term encompassing both themes and extensions. So an extension isn't a theme. You can see that in the values for the (unfortunately named) `ExtensionInfo`, one of whose values is "extension" and another of whose values is "theme". For the same reason "theme extension" in the page reads weirdly to me.

Also this note uses the passive voice.

So all that said, I would write something like:

```
Using this method you can only install add-ons whose <a href="https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/management/ExtensionInfo"><code>management.ExtensionInfo.type</code></a> property is <code>theme</code>.
```

Or (better perhaps):

```
The installable file pointed to by <code>url</code> must be a <a href="https://developer.mozilla.org/docs/Mozilla/Add-ons/Themes">theme</a>, not a normal browser extension.
```

**Comment by irenesmith** [comment]
I updated the message to use your second example. It's not my favorite, but I understand the problem with the passive voice on my original commit.

**Comment by wbamberg** [comment]
This push a build failure. I think it might be happier with single-quotes for the href value and double quotes for the whole note, although I can't find that documented anywhere. Sorry I didn't actually test my example, I was just showing what it might look like.

As a general tip: try running `npm run lint` at the command line before committing and pushing changes, that will show you these errors.

File: webextensions/api/management.json (hunk 1 above)
Comment (wbamberg) [reply webextensions/api/management.json]: The rendered table will just omit those columns (at least in the WebExtension docs) like: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/downloads/download#Browser_compatibility.

**Pull request merged**